File: poulet-mecanique-frontend/src/app/data/feed-property.ts
export class FeedProperty {

    id: number;
    label: string;
    discordChannel: string;
    url: string;
    icon: string;
    filter: string;

    constructor(id: number, label: string, discordChannel: string, url: string, icon: string, filter: string) {
        this.id = id;
        this.label = label;
        this.discordChannel = discordChannel;
        this.url = url;
        this.icon = icon;
        this.filter = filter;
    }

}

File: poulet-mecanique-backend/src/main/java/fr/arkyan/popo/pouletmecaniquebackend/data/entity/FeedProperty.java
package fr.arkyan.popo.pouletmecaniquebackend.data.entity;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Data
public class FeedProperty {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column
    private String label;

    @Column
    private String discordChannel;

    @Column
    private String url;

    @Column
    private String icon;

    @Column
    private String filter;

}
